src/overlays/actors/ovl_Bg_Treemouth/z_bg_treemouth.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_bg_treemouth.h"
+#include "versions.h"
```

Comment:
not the PR to address this, but if anything truly belongs in `global.h`, surely it's `versions.h`, this has nothing whatsoever to do with the actor and is so silly it has to be explicitly included in the overlay like this.

Reply:
FWIW I agree, others would like to get rid of global.h entirely but I think that would lead to suffering. Relevant discussion: https://github.com/zeldaret/oot/pull/2099#discussion_r1732300882